TicketDesk/ngWebClient-StandAlone/src/app/contact-info/contact-info.component.ts

```diff
@@ -1,18 +1,23 @@
 import { AdContactService } from './../services/ad-contact.service';
 import { UserDetails } from 'app/models/user-details';
 import { Component, OnInit, Input } from '@angular/core';
+import { Router, ActivatedRoute } from '@angular/router';
+import { NUMBER_TYPE } from '@angular/compiler/src/output/output_ast';
```

Comment:
Not sure what this is being used for and it's making me kind of nervous

Reply:
removed